CRM/Upgrade/Incremental/php/SixZero.php

```diff
@@ -42,8 +42,8 @@ public function upgrade_6_0_alpha1($rev): void {
       "timestamp NULL DEFAULT NULL COMMENT 'Date this membership modification action was logged.'",
       FALSE
     );
-    $this->addTask('Set a default activity priority', 'addActivityPriorityDefault');
-    $this->addSimpleExtensionTask(ts('enable dedupe backward compatibility'), ['legacydedupefinder']);
+    $this->addTask(ts('Set a default activity priority'), 'addActivityPriorityDefault');
```

Comment:
Believe it or not these _shouldn't_ have ts, except for the runSQL one (which is also arguable but it always does). A general rule is that strings that aren't seen by users usually are not ts'd because:
* Admins will often grep or websearch the string. The english is more useful for that.
* To lighten the load on translators

And then also in the upgrade case they are usually one-off strings never seen again, so again to lighten the translation load. But this is why it's ok that runSQL is translated, because it's in every upgrade.

Ditto for SixOne.php

Reply:
Ah, thanks. Fixed.